src/metabase/events/notification.clj

```diff
@@ -52,11 +57,29 @@
   (when (supported-topics topic)
     (models.notification/notifications-for-event topic)))
 
+(defn- app-name-trs
+  "Return the user configured application name, or Metabase translated
+  via trs if a name isn't configured."
+  []
+  (or (public-settings/application-name)
+      (trs "Metabase")))
```

Comment:
Very curious what translations we have for this 🤔

Reply:
haha, well most are just Metabase, but some languages are translated, interestingly the transition in Vietnamese is basically "database". which is funny.